### DISCUSSION TITLE
UBL multiple meshes

### DISCUSSION THREAD

**Discussion by Cancklehead** [Jyers/Marlin#674] [Q&A] [answered]
I've uploaded the latest jyers Extensive UI and am trying out the UBL for the first time. I've used bilinear with the bltouch before without problems.
What I would like is to set 2 meshes for 2 different build plates. I can run ans save the first mesh, but can't save a second one. When I give G29 S1 Octoprint returns  EEPROM can hold 1 meshes. How can I enable more meshes?
Thanks alot.

**Comment by gothcha** [comment, marked as answer]
The message is correct there is only room for one large mesh in the EEPROM. See #488

**Reply by Jyers** [reply]
I'm looking into adding this for the 10x10 configs if there is enough eeprom space.
